### PR TITLE
Fix error during RpmITCase tests on Windows

### PR DESCRIPTION
**Pull request by victornoel**
This is for #86. We still need to wait for CI to pass to validate it's ok.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#115](https://github.com/artipie/rpm-adapter/issues/115) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/d3b0fb71-a83e-4960-bd72-1fc730df4eb4); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/artipie/rpm/RpmITCase.java [hunk 1]
```diff
@@ -48,10 +48,6 @@
  *  where SHA1 is a HEX from SHA1 of file content and TYPE is a type of file
  *  (primary, others, filelists). Don't forget to uncompress it first.
  *  repomd.xml is not compressed and stored at `repodata/repomd.xml`.
- * @todo #69:30min This test is failing on Windows build. It's failing because of error
- *  FileAlreadyExistException in PrimaryXml.close in Files.move, but it's using
- *  REPLACE_EXISTING options, so it should not fail and it's not failing on Linux.
- *  Let's discover the problem, fix it, and remove DisabledOnOs(OS.WINDOWS) annotation.
```
Comment (g4s8) [inline src/test/java/com/artipie/rpm/RpmITCase.java:54]: @victornoel it seems you forgot to remove `@DisabledOnOs(OS.WINDOWS)` annotation

Comment (victornoel) [reply src/test/java/com/artipie/rpm/RpmITCase.java:54]: @g4s8 arg, that's why it worked ^^

**Comment by victornoel** [comment]
@g4s8 should be alright, I had to reorganize a bit things to arrive to this conclusion, see the various commits

**Comment by victornoel** [comment]
@g4s8 I have fixed the conflicts with master

**Comment by victornoel** [comment]
@g4s8 problem is I'm not sure the tests are enabled in the CI since #118 was merged (but the fixed worked before that ; )

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) I see that you unassigned this issue; the order is still assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk) though; to cancel the order use `refuse`, as in [§6](http://www.zerocracy.com/policy.html#6)

**Comment by 0crat** [comment]
Job [#115](https://github.com/artipie/rpm-adapter/issues/115) is closed, can't start order

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
Job was finished in 31 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by sereshqua** [comment]
@0crat quality bad

**Comment by 0crat** [comment]
Quality is low, no payment, see [§31](http://www.zerocracy.com/policy.html#31)

**Comment by Vatavuk** [comment]
@sereshqua just to illustrate my point from previous PR. Here you verdict is `bad`, which is ok since I dind't post any comments, but this only impacts the reviewer payment since the developer will get full amount. The quality that you check in the PR and your end verdict only impacts the reviewer not a developer so the qa should not take into account developer mistakes.

**Comment by sereshqua** [comment]
@Vatavuk users have to start their comments with the name of other users their comments are referred to

feel free to submit policy change request issue in [this](https://github.com/zerocracy/zerocracy.github.io) repo

**Pull request merged**